exasol_script_languages_container_ci/lib/ci_test.py

```diff
@@ -5,15 +5,15 @@
 from exasol_script_languages_container_tool.cli.commands import run_db_test
 
 import exasol_script_languages_container_ci
-from exasol_script_languages_container_ci.lib import print_docker_images
+from exasol_script_languages_container_ci.lib.common import print_docker_images
 
 
 def ci_test(ctx: click.Context, flavor_path: Tuple[str, ...]):
     """
     Run db tests
     """
 
-    if "skip tests" not in exasol_script_languages_container_ci.lib.get_last_commit_message():
+    if "skip tests" not in exasol_script_languages_container_ci.lib.common.get_last_commit_message():
         logging.info(f"Running command 'run_db_test' with parameters {locals()}")
```

Comment:
I think, we should ignore skip tests for master/Main and release

Reply:
Done

Reply:
Also for develop we should ignore skip tests

Reply:
We should extract the evaluation of skip tests to its own function for easier testing